README.md

```diff
@@ -2,53 +2,76 @@
 
-What we'll do to the app:
-- Make a request to the server with a name and get back a HTTP 200 response
+The goal of this tutorial is to show you what a basic, 'hello-world' Loopback.next app looks like and how it works.
 
-## Setup
+This example will show you how to: 
 
-[CLEANUP]
-Before you clone ... 
-### Install `node`, `npm`, and `TypeScript` 
-You should have `node` (>= 7.6.0), `npm` (>= 3.10), and `TypeScript`  (>= 2.0.0) installed. For the latest information on project dependencies for LoopBack, please check this link: 
-https://github.com/strongloop/loopback-next/wiki/Installation
+* create a Loopback.next server
+* make a request to the server to fetch content
 
-```
-git clone https://github.com/strongloop/loopback-next-hello-world.git
-cd loopback-next-hello-world
+This example will explain:
+
+* the structure of a Loopback.next application
+* how to add a custom component (TK)
+* how add a custom sequence (TK)
+
+When you have this example installed and running, you will be able to make a request to the server with a name and get back a HTTP 200 response: 
+
+[TODO: add screenshot of output here]
+
+## <a name="setup"></a>Setup 
+
+1. Install `node`, `npm`, and `TypeScript` 
+
+	You should have `node` (>= 7.6.0), `npm` (>= 3.10), and `TypeScript`  (>= 2.0.0) installed. 
+	
+	For the latest information on project dependencies for LoopBack, please check this link: 
+[https://github.com/strongloop/loopback-next/wiki/Installation](https://github.com/strongloop/loopback-next/wiki/Installation).
+
+2. Clone this repo: 
+	
+	`git clone https://github.com/strongloop/loopback-next-hello-world.git`
+
+3. Switch to your new directory: 
+	
+	`cd loopback-next-hello-world`
+
+4. Install your dependencies: 
+
+	```
 npm install -S @loopback/core
 npm install
-npm start
-# On another terminal, run the following curl command
-curl localhost:3000/helloworld?name=Joe
-# you should see "Hello world Joe!"
 ```
 
-> Note: `npm start` automatically builds the project via the `prestart` NPM script in package.json
+5. Start the application! 
+	
+	`npm start`
 
-## Useful app commands
+> Note: `npm start` automatically builds the project via the `prestart` npm script in `package.json`. If you would like to build this project separately, use `npm run build`.
 
-- Clone this project: `git clone https://github.com/strongloop/loopback-next-hello-world.git`
-- Building this project: `npm run build`
-- Running the application: `npm start`
-- Test your application: `npm test`
 
-## Project structure
+## <a name="testing"></a>Testing
+
+To test this application use `npm test`.
+
+## <a name="structure"></a>Project structure
 
 A TypeScript project will have separate `src` and `dist` directories, for source and distributable files. TypeScript (.ts) files from your `src` folder are compiled into JavaScript (.js) files and output in the `dist` folder.  
```

Comment:
Once merged @emckean will work on README.  I've communicated this point with @emckean .